ENH `learning_curve` raises a warning on failure of CV folds (#26299)
scikit-learn/scikit-learn: doc/whats_new/v1.3.rst

  `return_indices` to return the train-test indices of each cv split.
  :pr:`25659` by :user:`Guillaume Lemaitre <glemaitre>`.

> - |Enhancement| :func:`model_selection.learning_curve` raises a warning on
>   failure of the `cv` folds. :pr:`26299` by :user:`Rahil Parikh <rprkh>`.

Reviewer: ```suggestion
- |Enhancement| :func:`model_selection.learning_curve` raises a warning when
  every cross validation fold fails. :pr:`26299` by :user:`Rahil Parikh <rprkh>`.
```
